config.py

```diff
@@ -131,6 +134,10 @@ def get_parsed_args():
                                 'profile': False}), []
     return options, args
 
+def a7_compatible_to_int(status):
+    if status == A7_COMPATIBILITY_READY:
+        return 1
+    return 0
```

Comment:
nit: since this is only used in `collector.py`, can we move it there and spare the import?